packages/account/src/Components/account-limits/account-limits.tsx

```diff
@@ -232,116 +236,14 @@ const AccountLimits = observer(
-                                                            <AccountLimitsTableCell>
-                                                                <Localize i18n_default_text='Total withdrawn' />
-                                                            </AccountLimitsTableCell>
-                                                            <AccountLimitsTableCell align='right'>
-                                                                {formatMoney(
-                                                                    currency,
-                                                                    withdrawal_since_inception_monetary,
-                                                                    true
-                                                                )}
-                                                            </AccountLimitsTableCell>
-                                                        </tr>
-                                                        <tr>
-                                                            <AccountLimitsTableCell>
-                                                                <Localize i18n_default_text='Maximum withdrawal remaining' />
-                                                            </AccountLimitsTableCell>
-                                                            <AccountLimitsTableCell align='right'>
-                                                                {formatMoney(currency, remainder, true)}
-                                                            </AccountLimitsTableCell>
-                                                        </tr>
-                                                    </React.Fragment>
-                                                )}
-                                            </tbody>
-                                        </table>
-                                        {(!is_appstore || isMobile()) && (
-                                            <div className='da-account-limits__text-container'>
-                                                <Text as='p' size='xxs' color='less-prominent' line_height='xs'>
-                                                    {is_fully_authenticated ? (
-                                                        <Localize i18n_default_text='Your account is fully authenticated and your withdrawal limits have been lifted.' />
-                                                    ) : (
-                                                        <Localize i18n_default_text='Stated limits are subject to change without prior notice.' />
-                                                    )}
-                                                </Text>
-                                            </div>
-                                        )}
-                                    </React.Fragment>
-                                )}
+
+                                <WithdrawalLimitsTable
```

Comment:
Here we are rendering WithdrawalLimitsTable component, and then WithdrawalLimitsTable returns null if "is_app_settings" isn't true. Can we simplify this by checking the condition directly here?

Reply:
@amina-deriv, adding more checks to this function causes it's cognitive complexity to increase, that's why I had to extract most of those conditional checks out. But I can put it back if that's what's desired